route53_crossfeed_app.tf

```diff
@@ -374,6 +374,18 @@ resource "aws_route53_record" "crossfeed_staging_cd_api_CNAME" {
   zone_id = aws_route53_zone.cyber_dhs_gov.zone_id
 }
 
+resource "aws_route53_record" "crossfeed_staging_api_acme_TXT" {
+  provider = aws.route53resourcechange
+
+  name = "_acme-challenge.api.staging.crossfeed.${aws_route53_zone.cyber_dhs_gov.name}"
```

Comment:
@lfilip-ivaal did we also need a TXT record for staging.crossfeed? Or only needed one for api.staging.crossfeed?